protos/network.proto

```diff
@@ -46,3 +46,24 @@ message NetworkAcknowledgement {
 
     Status status = 1;
 }
+
+message GossipBlockRequest {
+    // The id of the block that is being requested
+    string block_id = 1;
+
+    // The identity of the validator that is requesting the block
+    bytes node_id = 2;
+
+}
+
+message GossipBatchRequest {
+    // Need to clarify if the id belongs to a batch or to a transaction
+    string id_type = 1;
```

Comment:
Should this be an enum?

It seems like it could be two different requests - GossipBatchByBatchIdRequest, GossipBatchByTransactionIdRequest.

In the case where we are requesting based on transaction ids, it seems like sending one at a time will be very chatty, since 1000 txns could all be in the same batch.